doc/safety.md

```diff
@@ -0,0 +1,57 @@
+# SECURITY
+
+This document describes the security recommendations and issues that can be helpful for users of Bitcoin Core.
+
+
+Download Bitcoin Core only from below links, verify the integrity of the download before using:
+
+https://bitcoincore.org/en/download/
+
+https://github.com/bitcoin/bitcoin/releases
+
+Downloading Bitcoin Core from untrusted sources can result in using a [malware](https://bitcoin.stackexchange.com/a/107738/) that can steal your bitcoin and compromise your computer.
```

Comment:
Maybe explicitly mention where to find official RCs, and caution that any solicitation for private testing is likely malware (though we want to be careful not to dissuade users from heloing actually debug issues, so how it's phrased should consider that too)

Reply:
Sure. Will add more information today.

Reply:
Done in `088d99b5e5c9bfd907861ce68692e638ee56448d`